src/main/kotlin/no/item/xp/plugin/GenerateTypeScriptTask.kt

```diff
@@ -88,10 +88,13 @@ open class GenerateTypeScriptTask @Inject constructor(private val extension: Gen
     )
   }
 
-  private fun handleSuccess(sequence: Sequence<Option<GeneratedField>>) {
+  private fun handleSuccess(sequence: Sequence<Option<Any>>) {
+    // sende sequencen til writer'n
```

Comment:
Don't commit these comments. If you need to tell the reader of the code something, put it in the String in the `TODO`.

Reply:
removed